integrationTests/relayers/slowTests/framework/multiversxHandler.go

```diff
@@ -1043,6 +1066,33 @@ func (handler *MultiversxHandler) TransferToken(ctx context.Context, source Keys
 		"hash", hash, "status", txResult.Status)
 }
 
+func (handler *MultiversxHandler) GetTotalBalancesForToken(ctx context.Context, token string) *big.Int {
+	queryParams := []string{
+		hex.EncodeToString([]byte(token)),
+	}
+	responseData := handler.ChainSimulator.ExecuteVMQuery(ctx, handler.SafeAddress, getTotalBalances, queryParams)
+	value := big.NewInt(0).SetBytes(responseData[0])
```

Comment:
is there any chance that responseData is nil here, resulting in a panic while trying to access index 0?